src/app/profile/email-verification/email-verification.component.html

```diff
@@ -0,0 +1,33 @@
+<div  class="container f8-email-verify">
+  <div class="row">
+    <div class="blank-slate-pf">
+      <div class="rh-branding-logo">
+        <img class="width-100"
+          src="../../../assets/images/Logotype_RH_OpenShift-io_RGB_RedGray.png">
+      </div>
+      <div *ngIf="verified === 'true'">
+        <span class="f8-email-verify-img img-trophy"></span>
```

Comment:
Would it make sense to just have this be an `<img>` with the `.f8-email-verify-img` class on it? Not sure the use of background-image is required here? I could be wrong though :)